pyaedt/hfss.py

```diff
@@ -55,37 +55,40 @@ class Hfss(FieldAnalysis3D, object):
     Examples
     --------
 
-    Create an instance of HFSS and connect to an existing HFSS
-    design or create a new HFSS design if one does not exist.
+    Create an instance of `HFSS` and connect to an existing HFSS
+    design or create a new `HFSS` design if one does not exist.
 
     >>> from pyaedt import Hfss
-    >>> hfss = Hfss()
+    >>> hfss2 = Hfss()
 
-    Create an instance of HFSS and link to a project named
-    ``projectname``. If this project does not exist, create one with
+    Create an instance of `HFSS` and link to a project named
+    ``HfssProject``. If this project does not exist, create one with
     this name.
 
-    >>> hfss = Hfss(projectname)
+    >>> hfss = Hfss("HfssProject")
+    pyaedt Info: Added design 'HFSS_...' of type HFSS.
 
-    Create an instance of HFSS and link to a design named
-    ``"designname"`` in a project named ``"projectname"``.
+    Create an instance of `HFSS` and link to a design named
+    ``HfssDesign1`` in a project named ``HfssProject``.
 
-    >>> hfss = Hfss(projectname,designame)
+    >>> hfss2 = Hfss("HfssProject","HfssDesign1")
+    pyaedt Info: Added design 'HfssDesign1' of type HFSS.
 
-    Create an instance of HFSS and open the specified project,
+    Create an instance of `HFSS` and open the specified project,
     which is named ``"myfile.aedt"``.
 
-    >>> hfss = Hfss("myfile.aedt")
+    >>> hfss2 = Hfss("myfile.aedt")
+    pyaedt Info: Added design 'HFSS_...' of type HFSS.
 
-    Create an instance of HFSS using the 2021 R1 release and open
+    Create an instance of `HFSS` using the 2021 R1 release and open
     the specified project, which is named ``"myfile.aedt"``.
 
-    >>> hfss = Hfss(specified_version="2021.1", projectname="myfile.aedt")
+    >>> hfss2 = Hfss(specified_version="2021.1", projectname="myfile.aedt")
 
-    Create an instance of HFSS using the 2021 R2 student version and open
+    Create an instance of `HFSS` using the 2021 R2 student version and open
     the specified project, which is named ``"myfile.aedt"``.
 
-    >>> hfss = Hfss(specified_version="2021.2", projectname="myfile.aedt", student_version=True)
+    >>> hfss2 = Hfss(specified_version="2021.2", projectname="myfile.aedt", student_version=True)
```

Comment:
Out of curiosity, why is this `hfss2` instead of `hfss`?

Reply:
Great catch, it was just a temporary to fix a former hfss test/example issue (I fixed it.). It should definitely be ``hfss``.